### DISCUSSION TITLE
Failed Initialization of LoRa-E5 using SX126x code

### DISCUSSION THREAD

**Discussion by aaseagrass** [jgromes/RadioLib#635] [Q&A]
Hello, I'm currently working with Grove LoRa-E5 modules. I tried to use SX126x code to initialize the LoRa-E5 but it failed. Could someone help me with this?

**Comment by jgromes** [comment]
That module is using an STM32WL - a microcontroller with integrated SX126x transceiver. It's currently not supported, however, work for that is ongoing here:  #588.
